domains/abdulrshaikh.json

```
@@ -0,0 +1,12 @@
+{
+    "description": "Abdul's personal developer website",
+    "repo": "https://github.com/ariskycode/ariskycode.github.io",
+    "owner": {
+        "username": "ariskycode",
+        "email": "arirfanshaikh23@gmail.com"
+    },
+    "record": {
+        "CNAME": "https://ariskycode.github.io",
```

CNAME records cannot be a full url. It has to be a host name.

```suggestion
        "CNAME": "ariskycode.github.io",
```